#!/usr/bin/env python

PACKAGE="attitude_estimation"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("GyroPitchRateStdDev", double_t, 0, "std dev of gyro noise ", 0.0065, 0.0, 0.01)
gen.add("GyroPitchRateBiasStdDev", double_t, 0, "std dev of gyro bias noise", 0.0005, 0.0, 0.01)
gen.add("AcceleroPitchStdDev", double_t, 0, "std dev of aceelero noise", 0.01, 0.0, 0.5)


exit(gen.generate(PACKAGE, "attitude_estimation","KFPitchCFG"))
